packages/host/tests/integration/components/card-basics-test.gts

```diff
@@ -2283,6 +2285,157 @@ module('Integration | card-basics', function (hooks) {
       await waitUntil(() => cleanWhiteSpace(root.textContent!) === 'Quint');
     });
 
+    test('Re-order linksToMany fields action will use the cached template that UI state', async function (assert) {
```

Comment:
Is there a word missing here? I don’t understand the name

Reply:
`will use the cached template OF that UI state`??? 🤔 Maybe there's a better way to phrase